src/backend/distributed/operations/repair_shards.c

```diff
@@ -700,6 +700,11 @@ CopyShardTables(List *shardIntervalList, char *sourceNodeName, int32 sourceNodeP
 {
 	ShardInterval *shardInterval = NULL;
 
+	MemoryContext localContext = AllocSetContextCreate(CurrentMemoryContext,
+													   "CopyShardTableLoop",
```

Comment:
We don't have a consistent naming strategy so far, but since this concerns more than a loop, maybe just name it "CopyShardTables" after the function.